docs/deployment/sample_privacy_policy.rst

```diff
@@ -27,13 +27,11 @@ Collection of Information From Sources
   message is automatically deleted.
 
 * Journalists decrypt and read each message offline. They are encouraged to
-  delete messages from the server on a regular basis. The date and time of any
-  message will be securely deleted from the server when the message is deleted.
+  delete messages from the server on a regular basis
```

Comment:
nit: missing a period. 
optional change: it would be good to speak to the secure deletion part here. "Messages will be securely deleted from the server." or similar.

Reply:
Fixed. I've left the optional change for now to land this fix; might want to kick "secure delete" language around a bit more given different hardware configurations (SSD vs. non-SSD) but it's possible I'm overthinking it. :)